docs/en/observability/uptime-set-up-app.asciidoc

```diff
@@ -0,0 +1,168 @@
+[[uptime-set-up-choose-agent]]
+= Use the {uptime-app}
+
+beta[]
+
+You can now create monitors directly in the {uptime-app}.
+To try out this beta functionality, first you will need to enable Monitor Management:
+
+. Go to **{observability}** -> **Uptime**.
+. Click **Monitor Management**.
+. Review the terms and click **Enable** to start using Monitor Management.
+
+From here, you can either <<global-managed-testing-infrastructure,use global managed testing infrastructure>>
+or <<private-locations,create a private location>> to run monitors from your own premises.
+
+NOTE: To use the Elastic Synthetics integration, see the https://www.elastic.co/guide/en/observability/8.3/uptime-set-up.html#uptime-set-up-choose-agent[8.3 documentation]. The Elastic Synthetics integrations is similar to the {uptime-app} approach, but contains fewer configuration options.
+
+[discrete]
+[[global-managed-testing-infrastructure]]
+== Use global managed testing infrastructure
+
+// Is there better language we can use to describe what it is
+// instead of what is _not_? 
+With Elastic's global managed testing infrastructure, you can create and run monitors in multiple
+locations using the {uptime-app} the UI without having to set up {fleet},
+pull a Docker image, and install and run an agent on your machine.
+
+To add a monitor using Elastic's global managed testing infrastructure:
+
+. Go to the **Monitor Management** page.
+. Click **Add monitor**.
+. In _Monitor locations_, select one or more locations.
+. Choose a _Monitor type_ and configure the monitor as needed.
+. Click **Save monitor**.
+
+If you are setting up a **Browser** synthetic monitor, see <<synthetics-create-test>>
+for more details on writing synthetic tests.
+
+[discrete]
+[[private-locations]]
+== Run a monitor on private locations
+
+Private locations allow you to run monitors from your own premises in addition to or
+instead of on Elastic's global managed testing infrastructure.
+
+// The TL;DR of what follows
+Before you create a private location, you need to set up {fleet} and {agent},
+pull a Docker image, and connect to the {stack}. Then, you can create a private location
+in the {uptime-app} UI so when you add a new monitor, the private location will be
+available in the list of _Monitor locations_.
+
+[discrete]
+[[private-locations-fleet-agent]]
+=== Set up {fleet-server} and {agent}
+
+// Is this sufficient or should we copy steps for setting up
+// Fleet and Agent into this page directly?
```

Comment:
I'd say linking to that page is fine. If people are using private locations they're probably more advanced users anyway (or at least like the extra "freedom") so I don't think they would mind just a link.